lib/xcp_service.ml

```diff
@@ -360,6 +360,7 @@ type ('a, 'b) error = [
 
 let configure2 ~name ~version ~doc ?(options=[]) ?(resources=[]) () =
   try
+    Xcp_coverage.init name;
```

Comment:
So I take it that xapi must not call `configure2`.

Reply:
Yes, or maybe we should just separate the two init calls so you can call them both? And have XAPI just call them both explicitly.
Then if we want to switch XAPI to `Xcp_service.configure2` in the future we can do so without being hindered by this limitation.

Reply:
I'd make this change as well before merging

Reply:
I've pushed a commit to remove the init call from inside dispatcher_init, so we can do the separe calls as discussed above.